packages/libs/coreui/src/components/Mesa/types.ts

```diff
@@ -35,33 +35,48 @@ export interface MesaStateProps<Row, Key extends string> {
     selectedNoun?: string;
     selectedPluralNoun?: string;
     searchPlaceholder?: string;
-    deriveRowClassName?: unknown; // a function
-    renderEmptyState?: unknown; // a function
-    isRowSelected?: unknown; // a callback function
+    deriveRowClassName?: (row: Row) => string | undefined; // a function
+    renderEmptyState?: () => ReactNode; // a function
+    isRowSelected?: (row: Row) => boolean; // a callback function
```

Comment:
Can we remove these comments here and elsewhere now that the types are more clear?